recipe/meta.yaml

```
@@ -28,23 +28,24 @@ requirements:
     - blinker >=1.0.0,<2
     - cachetools >=4.0,<6
     - click >=7.0,<9
-    - numpy >=1.20,<3
-    - packaging >=20,<25
+    - numpy >=1.19.3,<3
```

If I am looking at the right place, I think the upper bound on numpy is 2 upstream:
https://github.com/streamlit/streamlit/blob/1.35.0/lib/setup.py#L36

```suggestion
    - numpy >=1.19.3,<2
```

---

True, thanks!